.github/workflows/bump.yml

```
@@ -0,0 +1,51 @@
+name: Bump
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseType:
+        description: 'Release type'
+        type: choice
+        required: true
+        default: patch
+        options:
+          - patch
+          - minor
+          - major
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: v20
+
+      - name: Get latest version
+        id: latest
+        run: |
+          LATEST=$(gh release list \
+            --exclude-drafts \
+            --exclude-pre-releases \
+            --json name,isLatest \
+            --jq '.[] | select(.isLatest == true) | .name')
+
+          echo "version=$LATEST" >> "$GITHUB_OUTPUT"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get next version 
+        id: next
+        run: |
+          NEXT=$(npx --yes semver -i ${{ github.event.inputs.releaseType }} ${{ steps.latest.outputs.version }})
+
+          echo "version=$NEXT" >> "$GITHUB_OUTPUT"
+
+      - name: Generate release
```

(optional) Would it be any safer to add a check here to make sure that `steps.next.outputs.version` is set?
```suggestion
      - name: Generate release
         if: ${{ steps.next.outputs.version != '' }}
```

Technically speaking, this step should never run if the previous fails anyway. So this may not really add to any safety.

---

Better safe than sorry. And it does make the dependency explicit in case the workflow changes in the future.